named.go

```diff
@@ -224,7 +224,7 @@ func bindStruct(bindType int, query string, arg interface{}, m *reflectx.Mapper)
 	return bound, arglist, nil
 }
 
-var valuesReg = regexp.MustCompile(`\)\s*(?i)VALUES\s*\(`)
+var valuesReg = regexp.MustCompile(`[\)|\(]\s*(?i)VALUES\s*\(`)
```

Comment:
I see test cases justifying `)` and `(` but not `|`. Unless anyone wants to add a test demonstrating the usefulness of allowing a leading `|`, maybe this PR could get unblocked by removing that option?

```suggestion
var valuesReg = regexp.MustCompile(`[)(]\s*(?i)VALUES\s*\(`)
```

Note there is no need to escape parentheses within a character class, since capture groups cannot exist inside character classes.